### ISSUE TITLE
Irish/BT postcodes GPS different to Google/Bing Map results

### ISSUE THREAD

**Issue by caihong** [ideal-postcodes/postcodes.io#36]
Great product and service. However the GPS results on Northern Ireland BT postcodes e.g. BT41 3UB are significantly different to those results from Google/Bing map search and other sites like http://irish.gridreferencefinder.com/, http://www.doogal.co.uk/ShowMap.php?postcode=BT41%203UB

e.g. the result from postcodes.io for BT41 3UB is
"longitude":-5.81387222553498,"latitude":54.8566885377204,

while the results from other sites are around the following point, which is about 40km's away.
lat:54.778698; lon:-6.437674

Appreciate that the postcodes.io lat,lon is likely calculated based on the irish grid reference provided in the ONSPD. Is this a problem at the source or the algorism?

Cheers!

**Comment by cblanc** [comment]
Thanks for bringing this up. Your understanding of the Irish Northings/Easting predicament is correct.

I think this is an issue with the source, i.e. a faulty data point from the ONS. I checked other ONS based sources, I retrieved the same geolocation as postcodes.io:
http://mapit.mysociety.org/postcode/BT41%203UB%20.html
http://www.uk-postcodes.com/postcode/BT413UB.html

I also checked similar postcodes on a map (postcodes.io/explore) searching for substring BT413, you can see all postcodes but BT413UB are clustered around the geolocation specified by Google/Bing maps.

Lastly, ran a quick search on Royal Mail's addressing database and the premises listed only make sense for Google/Bing's location

So in short, I think this is an error at source. I'll send an email and hopefully it should be fixed in the next edition.

**Comment by cblanc** [comment]
FYI just received confirmation that Land Property Services (Northern Ireland) have been notified. So hopefully this will go into the next edition
